frontends/election-manager/src/utils/election.test.ts

```diff
@@ -16,3 +18,77 @@ test('getBallotPath allows digits in file names', () => {
     'election-d34db33f-precinct-north-springfield-id-21-style-77-English-live-absentee.pdf'
   );
 });
+
+describe('generateOvervoteBallot adds an overvote conflict when possible', () => {
+  const precinctId = electionSample.precincts[0].id;
+
+  test('minimally overvotes an initial candidate contest', () => {
+    const overvoteBallot = generateOvervoteBallot({
+      election: electionSample,
+      precinctId,
+    });
+    expect(overvoteBallot).toBeDefined();
+
+    const votes = overvoteBallot!.votes as VotesDict;
+    const overvoteContestId = electionSample.contests[0].id;
+
+    expect(Object.keys(votes)).toEqual([overvoteContestId]);
+    expect(votes[overvoteContestId]?.length).toEqual(2);
```

Comment:
Did you consider boiling this down to a single assertion of what's in votes?

```test
expect(votes).toEqual(/* Exactly what's in votes */)
```

I can see arguments in favor of both approaches! The current approach makes the qualities of the state you want clear (e.g. 2 votes). The only reason I lean toward the single exact assertion is that the current code could pass if we overvoted a yes-no contest instead of a candidate contest since `['yes', 'no']` is also of length `2`